extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -308,6 +308,11 @@ public function getWikiVariables() {
 			$wikiVariables['facebookAppId'] = $egFacebookAppId;
 		}
 
+		$wikiImages = (new WikiService())->getWikiImages( [$this->wg->CityId], 500 );
+		if ( is_array( $wikiImages ) && is_string( $wikiImages[$this->wg->CityId] ) ) {
```

Comment:
Is it important that result be a string (or rather, is it possible it would be something else)?  I would use ```!empty``` here which will take care of checking if this is an array at whether there is anything at that array position at the same time, e.g.:
```php
if ( !empty( $wikiImages[$this->wg->CityId] ) ) {
```